capsules/detector_text_openvino/meta.conf

```diff
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.0
```

Comment:
Change this to 0.2
this is when the BaseOpenVINOBackend features were introduced.

Reply:
Also, add new line at end of file